### DISCUSSION TITLE
Does OTOBO always contact https://portal.rother-oss.com/otobo/public.pl

### DISCUSSION THREAD

**Discussion by escix** [RotherOSS/otobo#3574] [Q&A]
Does otobo always contact https://portal.rother-oss.com/otobo/public.pl even if news in the dashboard is disabled?
We run our instance in a vm with no web access due to security and privacy and the log files fill up with this error.

Also does otobo contact any other website/links?

**Comment by stefanhaerter** [comment]
How did you disable the news widget?

Another setting which affects communication is `CloudServices::Disabled`, perhaps you want to disable this.

**Reply by escix** [reply]
in the dashboard where you can select which widget to load.

Will try, thanks.

**Reply by stefanhaerter** [reply]
Do the error messages persist if you disable the sysconfig setting `DashboardBackend###0405-News`?

**Comment by stefanhaerter** [comment]
Nothing new here for quite some time, closing the discussion.
